core/controllers/android.py

```diff
@@ -545,7 +518,10 @@ class AndroidPlatformParametersHandler(
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
     URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
-    HANDLER_ARGS_SCHEMAS: Dict[str, Dict[str, HandlerArgEntryInt]] = {
+    # Here we use object because `HANDLER_ARGS_SCHEMAS` may contain
```

Comment:
In this case, it won't; the structure is fairly well determined. Don't use object, please use the exact type (it might be "A or B or C" if there are a few options).